contrib/docker/solr/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir -p /opt/solr/server/solr/$SOLR_CORE/data
 
 # Adding Files
 ADD ./solrconfig.xml \
-https://raw.githubusercontent.com/ckan/ckan/ckan-2.5.2/ckan/config/solr/schema.xml \
+https://raw.githubusercontent.com/ckan/ckan/master/ckan/config/solr/schema.xml \
```

Comment:
If we can get ckan using the ManagedSchema API we don't need the schema.xml in our solr image and users can start solr without additional configuration (a big chunk of our docs are devoted to this) but, it's a fair bit of work because some sites extend the schema in their own ways, so we might need a new interface to let plugins configure solr.